src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -698,6 +704,10 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, const pal
     }
     
     trace::verbose(_X("It was not possible to find any SDK version"));
+	if (cli_version_specified && !cli_version_found)
+	{
+	    trace::error(_X("The specified SDK version [%s] from global.json [%s] doesn't exist; install specified SDK version [%s]"), cli_version.c_str(), global.c_str(), cli_version.c_str());
```

Comment:
I think we should say the SDK was not "not found" rather than that it "doesn't exist".